website/content/docs/agent-and-proxy/agent/generate-config/index.mdx

```diff
@@ -75,7 +75,7 @@ auto_auth {
     type = "token_file"
 
     config {
-      token_file_path = "/Users/avean/.vault-token"
+      token_file_path = "/Users/avean/.openbao-token"
```

Comment:
Let's confirm that the token name has changed, and if it is "openbao" or just "bao"

Reply:
So far this hasn't changed, we can use what we want:

https://github.com/cipherboy/openbao/blob/8dead566a8ab017c97a6417df33d326167eb5f57/command/token/helper_internal.go#L35-L39

Using openbao might be better for clarity, IMO?

Reply:
I'd be in favor of `.openbao-token` even though the CLI is named `bao`. But I think for now I need to revert that change because the token path does not seem configurable in many cases. Also there's the question if we want to keep supporting `.vault-token` files at least for some time to ease adoption. Same goes for `X-Vault` HTTP headers, but that's a discussion for another time :-)